gradle/upload-shadow.gradle

```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ * Configuration to upload JAR file generated by shadowJar.
+ *
+ */
+
+apply plugin: 'java'
+apply plugin: 'maven'
+
+uploadShadow {
+    repositories {
+        mavenDeployer {
+            pom.groupId = GROUP
+            pom.artifactId = POM_ARTIFACT_ID
+            pom.version = VERSION_NAME
+
+            if ("$VERSION_NAME".contains("SNAPSHOT"))
```

Comment:
Yeees 🎉 that is an elegant way to distribute our dependencies. Awesome @rachelcarmena